pkg/features/kube_features.go

```diff
@@ -309,6 +309,12 @@ const (
 	// Enables support of HPA scaling to zero pods when an object or custom metric is configured.
 	HPAScaleToZero featuregate.Feature = "HPAScaleToZero"
 
+	// owner: @omerap12,@adrianmoisey
+	// kep: https://kep.k8s.io/5325
+
+	// Improve pod selection accuracy across workload types
+	HPAselectionStrategy featuregate.Feature = "HPAselectionStrategy"
```

Comment:
I'm not too sure on this nit: 
```suggestion
	HPASelectionStrategy featuregate.Feature = "HPASelectionStrategy"
```

But it matches the one above, "HPAScaleToZero" now

Reply:
Yeah I thought the one above is a mistake but I don't mind changing it to align with the others